src/main/java/thePackmaster/powers/rimworldpack/CatharsisPower.java

```diff
@@ -56,6 +60,10 @@ public void updateDescription() {
         else
             description = DESCRIPTIONS[1]+ amount + DESCRIPTIONS[2];
     }
+    @Override
+    public void atStartOfTurn() {
+        addToTop(new RemoveSpecificPowerAction(owner, owner, this));
```

Comment:
```suggestion
        addToBot(new RemoveSpecificPowerAction(owner, owner, this));
```

Reply:
This is the way Burst, etc do it. Might want to follow their example so its consistent